hamocc/mo_aufr_bgc.F90

```diff
@@ -0,0 +1,581 @@
+                                  isco213,isco214,izoo13,izoo14,safediv,                           &
+                                  issso13,issso14,isssc13,isssc14,ipowc13,ipowc14,                 &
+                                  iatmc13,iatmc14,iatmnco2,inatalkali,inatcalc,inatsco212,         &
+                                  ipowaal,ipowaic,ipowaox,ipowaph,ipowasi,ipown2,ipowno3,          &
+                                  isssc12,issso12,issssil,issster,ks
+    use mo_vgrid,           only: kbo
+    use mo_sedmnt,          only: sedhpl
+    use mo_intfcblom,       only: sedlay2,powtra2,burial2,atm2
+    use mo_param_bgc,       only: bifr13,bifr14,c14fac,re1312,re14to,prei13,prei14
+    use mo_read_netcdf_var, only: read_netcdf_var
+
+    ! Arguments
+    integer,          intent(in)    :: kpie                                              ! 1st dimension of model grid.
+    integer,          intent(in)    :: kpje                                              ! 2nd dimension of model grid.
+    integer,          intent(in)    :: kpke                                              ! 3rd (vertical) dimension of model grid.
+    integer,          intent(in)    :: ntr                                               ! number of tracers in tracer field
+    integer,          intent(in)    :: ntrbgc                                            ! number of biogechemical tracers in tracer field
+    integer,          intent(in)    :: itrbgc                                            ! start index for biogeochemical tracers in tracer field
+    real,             intent(inout) :: trc(1-nbdy:kpie+nbdy,1-nbdy:kpje+nbdy,2*kpke,ntr) ! initial/restart tracer field to be passed to the ocean model [mol/kg]
+    integer,          intent(in)    :: kplyear                                           ! year  in ocean restart date
+    integer,          intent(in)    :: kplmon                                            ! month in ocean restart date
+    integer,          intent(in)    :: kplday                                            ! day   in ocean restart date
+    real,             intent(in)    :: omask(kpie,kpje)                                  ! land/ocean mask
+    character(len=*), intent(in)    :: rstfnm                                            ! restart file name-informations
+
+    ! Local variables
+    real, allocatable :: locetra(:,:,:,:) ! local array for reading
+    integer   :: errstat
+    integer   :: restyear                          !  year of restart file
+    integer   :: restmonth                         !  month of restart file
+    integer   :: restday                           !  day of restart file
+    integer   :: restdtoce                         !  time step number from bgc ocean file
+    integer   :: idate(5),i,j,k
+    logical   :: lread_cfc,lread_nat,lread_iso,lread_atm,lread_bro
+    real      :: rco213,rco214,alpha14,beta13,beta14,d13c_atm,d14cat ! cisonew
```

Comment:
Please refrain from changing variable letters and revert back `d13c_atm` to `d13C_atm` (the capital C has a  clear meaning here). (at least that's my opinion)

Reply:
Sorry about that - this was inadvertent. I'll fix this!

Reply:
No worries, just commenting and going through your commit. Appreciate your work!

Reply:
Actually - looking at the original code - there are still a lot of places in the original code where d13c is lower case.
Should we change those?

Reply:
I guess, I leave this to @JorgSchwinger , in my opinion, it could be done, but since the isotopes are not my expertise thus far, I don't want to enforce here something. I just stumbled over it, when looking through your changes.